terraform/certs-ds/main.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
```

Comment:
Not sure if this was intentionally added or not?

Reply:
my bad. 👍

Reply:
thanks for catching it. already pushed a fix for it.